### ISSUE TITLE
BigSure support

### ISSUE THREAD

**Issue by yodagamaheshan** [bhagyas/osx-sinhalese-dictionary#4]
does not support for BigSure. even I added the dict file into dictionary folder manually.

**Comment by bhagyas** [comment]
It works on my testing environment for Big Sur (The dictionary works as intended), did you try to re-create the dictionary or did you upgrade from the earlier version?

**Comment by yodagamaheshan** [comment]
yes.I tried last time with Catalina, It didnt work even that time. 😔

**Comment by RubyAbey** [comment]
Hi, Does it support Monterey? Got this error.
<img width="274" alt="Screenshot 2022-08-30 at 15 06 07" src="https://user-images.githubusercontent.com/12810072/187444437-ee7c8173-2d19-48c2-ae58-64d44ac42386.png">

**Comment by bhagyas** [comment]
I have had some success with using `pyglossary` instead of `DictUnifier` to build  the dictionary. Using pyglossary, and Dictionary Development Kit from Apple, I was  able to build the following so you can skip them for now.

[english-sinhala.dictionary.zip](https://github.com/bhagyas/osx-sinhalese-dictionary/files/9455640/english-sinhala.dictionary.zip)

Extract and obtain the file `english-sinhala.dictionary` file and copy to `~/Library/Dictionaries` folder (This folder can sometimes be empty or non-existent). 

After the copy is completed, open the Dictionary.app and then goto Preferences and you should be able to see the english-sinhala dictionary. It will take a while  for the first few searches made on the Dictionary app or using Look Up feature, but any subsequent searches must be faster.
